src/website/src/app/app.component.ts

```diff
@@ -80,6 +80,17 @@ export class AppComponent implements OnInit {
     this.setTitle(filteredTitles.join(this.browserTitleSeparator));
   }
 
+  getDocumentationUrl() {
+    // If the URL starts with /documentation or isn't on clarity.design, just point to local version
+    if (window.location.pathname.indexOf('/documentation') === 0 || window.location.hostname !== 'clarity.design') {
+      return `/documentation`;
+    }
+    // Otherwise check if latest or not and redirect if not latest for prereleases
```

Comment:
Is there a fall through case? Or do these two conditionals cover all possible cases?

Reply:
Yeah I was testing just a few cases, but now can see the full logic and pushed update.